internal/operator/subscription.go

```diff
@@ -85,6 +85,23 @@ func (c operatorClient) DeleteSubscription(ctx context.Context, name string, nam
 	return c.Client.Delete(ctx, subscription)
 }
 
-func (c operatorClient) ListPackageManifests(ctx context.Context, list *pkgserverv1.PackageManifestList) error {
-	return c.Client.List(ctx, list)
+func (c operatorClient) ListPackageManifests(ctx context.Context, list *pkgserverv1.PackageManifestList, filter []string) error {
+	var tmppkgmlist pkgserverv1.PackageManifestList
+	if err := c.Client.List(ctx, &tmppkgmlist); err != nil {
+		return err
+	}
+
+	if len(filter) > 0 {
+		for _, f := range filter {
+			for _, pkgm := range tmppkgmlist.Items {
+				if pkgm.Name == f {
+					list.Items = append(list.Items, pkgm)
+				}
+			}
+		}
+	} else {
+		list.Items = tmppkgmlist.Items
```

Comment:
One thing that came to me is that if we should have some sort of output to tell the user that the package name provided doesn't exist on the desired catalog source. Not sure how relevant it is though.

Reply:
Yep I opened issues that are probably relevant to this but let me know if not; #116 and #117 @acmenezes

Reply:
Yes @mrhillsman , that works. I guess we can even address this in more than one PR if needed.

Reply:
@acmenezes yep I believe that is best as I did not want to make this PR too large and still need to work out the logic while filtering can be done for now in some capacity with this one.